dashboard/app/views/pd/workshop_user_admin/facilitator_courses_form.html.haml

```diff
@@ -0,0 +1,47 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+
+%h1 Manage Facilitator Courses
+
+#application-container.selectable
```

Comment:
I'm guessing you copied `.selectable` from [here](https://github.com/code-dot-org/code-dot-org/blob/f3eafebd01c7869828b0a0a26769e0a35ff29c68/dashboard/app/views/pd/workshop_material_orders/new.html.haml#L15), but that's actually a mistake leftover from an earlier iteration. The entire `#application-container` div is [selectable](https://github.com/code-dot-org/code-dot-org/blob/66ae78135de9e13e0ec475a315f7558ae70e27f1/apps/style/code-studio/pd.scss#L32) now, and the selectable class doesn't exist in our css. While you're removing that, I wouldn't mind if you remove it from that other spot too ;)